src/main/java/com/example/springboot/controller/RestaurantController.java

```diff
@@ -63,7 +63,7 @@
             return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
         }
     }
-    @PutMapping("rest/{id}")
+    @PutMapping("re/{id}")
```

Comment:
Add a single RequestMapping at the controller level as a best practice. You are exposing too many similar looking endpoints.